### ISSUE TITLE
Remove or fix img label computation tests

### ISSUE THREAD

**Issue by dandclark** [web-platform-tests/interop#648]
### Test List

In [accname/name/comp_label.html](https://wpt.fyi/results/accname/name/comp_label.html?label=experimental&label=master&aligned&view=interop&q=label%3Ainterop-2024-accessibility):
- aria-label undefined on img w/ alt
- aria-label undefined on img w/o alt	
- aria-label undefined on img w/ empty alt	
- aria-label undefined on img w/o alt but w/ title	
- aria-label undefined on img w/ empty alt but w/ title

### Rationale

These tests are meant to exercise the case in https://w3c.github.io/accname/#computation-steps where if the aria-label is undefined in step 2D, then the aria-label is not used and instead in step 2E the Host Language Label is used (`alt` or `title` in this case). However, the tests are setting the attribute to the literal string `"undefined"`, so it's by design that this value is used.

Additionally the last two tests suggest that the elements should have a `title` set but that is missing.

All engines currently fail these tests.

There is already a PR to remove these, although there's been no activity for a month: https://github.com/web-platform-tests/wpt/pull/44600

I think either the tests should be corrected, or the PR to remove them should be landed.

**Comment by zcorpan** [comment]
+1 to land https://github.com/web-platform-tests/wpt/pull/44600

cc @cookiecrook

**Comment by cookiecrook** [comment]
Yes
